core/src/main/java/jenkins/model/item_category/NestedProjectsCategory.java

```diff
@@ -0,0 +1,38 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import jenkins.model.Messages;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * A generic {@link ItemCategory}.
+ *
+ * This category should be moved to cloudbees-folder-plugin short-term.
+ */
+@Restricted(NoExternalUse.class)
+@Extension(ordinal = -100)
+public class NestedProjectsCategory extends ItemCategory {
+
+    public static final String ID = "nestedprojects";
```

Comment:
please add some trivial javadoc,
`/** The ID for {@link TopLevelItemDescriptor#getCategoryId} to use to be considered a Nested Project. */`